gc/stats/HeapResizeStats.hpp

```diff
@@ -173,7 +174,11 @@ class MM_HeapResizeStats : public MM_Base
 			return 0;
 		}			
 	}
-	
+
+	MMINLINE void resetExcludeCurrentGCTimeFromStats() { _excludeCurrentGCTimeFromStats = FALSE; }
+	MMINLINE void setExcludeCurrentGCTimeFromStats() { _excludeCurrentGCTimeFromStats = TRUE; }
```

Comment:
I won't be surprised if compile warnings (or errors) result from this (mixing `TRUE`/`FALSE` with `bool`); the assignments should use `true` / `false` (consistent with line 199 below).

Reply:
Thanks for catching this. Even if compilers tolerate this, we (@jason-hall) should clean it up.